Zend/zend_ast.c

```diff
@@ -476,7 +476,7 @@ static zend_result zend_ast_add_unpacked_element(zval *result, zval *expr) {
 	}
 
 	/* Objects or references cannot occur in a constant expression. */
```

Comment:
This comment is outdated for objects (since `new Foo()`).

Reply:
True. Removed.